### PR TITLE
Create website

### PR DESCRIPTION
**Pull request by samroodali**

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello @SamroodAli, I can see lots of effort in this project. You have done very well until now. Congratulation on that. 👏 👏 👏 

## The highlights: 
* Your `git-flow` is correct. 💯 
*  Setting up `linters` is correct. 💯 
*  The project is looking great, you followed all the basics of `HTML5&CSS` concepts. 👍 
*  Nice work on the `Navbar` part ⭐ 
*  Nice work on `embedded images and video`. ⭐ 

## What to do next:  

**Now let's work on some requirements, which need to complete in order to get approval.**
**Please remember all the checklist needs to add**

## [REQUIRED]

- [x]  Please add a meaningful title and a short description to the Pull Request.

**README.md**
- [x]  Add a screenshot of the project in the browser.
- [x]  Add a working live demo.
- [x]  In the `Getting Started` section add instructions for download and locally run your project.

**Design Part**
- [x] Elements aligned with a combination of all three techniques float, flex, grid. I can see flex and grid but you need to use [float](https://css-tricks.com/all-about-floats/) too.
- [x]  The `Editor's picks` section is not at the starting of the page, please reposition this. 
screencapture-file-F-TSE-new-york-times-article-clone-create-website-index-html-2021-02-17-21_23_43
You can check the real design [here](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

- [x]  The `The Theory of Inflation` section also needs to retouch.
ss2
Please compare the red marked area with the real design, and make the changes.

- [x] There is a border in the starting of the **Bottom part**, compare two images:

Your project 👇 
ss3

The main site 👇 
ss4


**In the bottom part**
- [x]  The `Most popular` has all the link components. 
- [x]  The `More in Space and Astronomy` part you need to add a margin between the articles.
- [x] The `SUBSCRIBE` section add some margin and padding.
- [x]  The Footer is missing in your design. Check the red arrow 👇 
ss5
- [x]  Go to Home Page » link is missing. check the previous image, check the red box. ☝️  


**Changes to make in CSS file**
- [x] Bad CSS Practice: You are using repeated code like `display: grid;` 16 times, `display: flex;` 7 times,  and many more. This is bad CSS practice also against the DRY rule. Instead of this, you can write a single class like `.grid { display: grid;  }`
- [x]  Remove the commented line 33 from the stylesheet.

## 

**OOOh!! That's a huge list!! But don't worry. It's normal** ☺️ 

**Please add all this to your project and then ask again for a CRR.**

Feel free to reach me on slack @lamiasristy 🙋

**Keep up the spirit always.** 

Happy Coding 💻

File: style.css [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+  line-height: 1em;
+  font-size: 0.9em;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+}
+
+.nav-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (LamiaSristy) [inline style.css]: You are using all these stylings many times differently, you can make a new class named `.flex-center` and use it.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,745 @@
...
-    <title>Document</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>
+        Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times
+    </title>
+    <!-- <link rel="stylesheet" href="style.css" /> -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
```
Comment (LamiaSristy) [inline index.html]: - [ ] It's a good practice to use `./ ` before the href. <link rel="stylesheet" href="./style.css">.
- [ ]  Remove the commented line 11.

**Comment by samroodali** [comment]
I have completed the requirements and I updated the readme file.
There was an issue with the alignment of the readme which I only fixed after submitting the pull request.
But it is fixed now.
Thank you @LamiaSristy for the code review, your requirements were very clear and I enjoyed improving my code.
I hope I have met all the requirements

**Review by GzimAsani — APPROVED** [review]
# Status: PROJECT APPROVED :+1: : :100:  
## Hello **Team** 👋, good job implementing all the requirements asked from the previous reviewer :muscle:, you have met all the requirements so go ahead and merge this pull request.

_Ps: Don't forget to submit the project completion form_  


### HAPPY CODING :computer:

**Comment by samroodali** [comment]
> # Status: PROJECT APPROVED 👍 : 💯
> ## Hello **Team** 👋, good job implementing all the requirements asked from the previous reviewer 💪, you have met all the requirements so go ahead and merge this pull request.
> _Ps: Don't forget to submit the project completion form_
> 
> ### HAPPY CODING 💻

Thank You

**Pull request merged**